src/core/Pricing/PricingCards.stories.tsx

```diff
@@ -0,0 +1,53 @@
+import React from "react";
+import PricingCards from "./PricingCards";
+import { consumptionData, planData } from "./data";
+
+export default {
+  title: "Features/Pricing Cards",
+  component: PricingCards,
+  tags: ["autodocs"],
+  parameters: {
+    docs: {
+      description: {
+        component:
+          "A reusable component that forms the basis of two sections of the pricing page. It is used to display pricing plans and consumption pricing. The presence of a `delimiter` prop changes the UX to focus around intermediate icon columns.",
+      },
+    },
+  },
+};
+
+export const PlansDarkMode = {
```

Comment:
It would be ace if I could leverage Storybook's dark mode plugin to change this in the manager instead of defining things twice, but I can't twist it into doing that